.moai/specs/SPEC-HIERARCHICAL-TEAM-001/progress.md

```
@@ -0,0 +1,52 @@
+M6: AC-CLOSE-001=PASS, AC-REGRESS-001=PASS | evidence: orchestration-mode-selection.md §G.2 added (manager-lead as Mode-5-shaped delegation target, NOT Mode 7; Mode 1-6 catalog unchanged; --mode dispatch-axis values unchanged; MODE_TEAM_UNAVAILABLE unchanged); moai spec lint --strict on spec.md = 0 errors / 0 warnings | fold-at: 2026-08-07T13:35:00Z
+
+## §E.3 Run-phase Audit-Ready Signal
+
+_(pending run-phase — manager-develop populates this section when all M1-M6 MUST ACs PASS.)_
+
+## §E.4 Sync-phase Audit-Ready Signal
+
+- `sync_status: completed`
+- `sync_complete_at: 2026-08-07` (ISO-8601 placeholder — manager-docs emission; actual timestamp is the sync commit's author date)
+- `sync_commit_sha: pending-backfill-SPEC-HIERARCHICAL-TEAM-001` (self-referential-hazard workaround per `spec-frontmatter-schema.md` D3; backfilled in a follow-up commit after the sync PR merges — same pattern as SPEC-NAVIGATOR-SYNC-003 / SPEC-MOAI-MCP-SERVER-001)
+- `changelog_entry_position: CHANGELOG.md `[Unreleased] → Added` (top of section, above SPEC-MOAI-MCP-SERVER-001)`
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)